src/apps/common/submodules/conferenceViewer/conferenceViewer.js

```diff
@@ -374,6 +375,9 @@ define(function(require) {
 
 					break;
 				case 'start-talking':
+					if ($userDiv.find('[data-action-category="mute"]').hasClass('active')) {
```

Comment:
Here you can reverse the test:
```suggestion
					if (!$userDiv.find('[data-action-category="mute"]').hasClass('active')) {
```